.github/workflows/publish.yml

```diff
@@ -58,3 +58,28 @@ jobs:
           helm-gcs-credentials: ${{ secrets.HELM_GCS_CREDENTIALS }}
           helm-gcs-repository: ${{ secrets.HELM_GCS_REPOSITORY }}
 
+  publish-release-notes:
+    needs: [ publish-artifacts, publish-helm-charts ]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - uses: lakto/gren-action@v2.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          options: '-c .grenrc.js --override'
+
+  publish-changelog:
```

Comment:
Do we need this step? If, yes, does someone has to create PR at the end?

Reply:
This was to generate `RELEASE_NOTES.md` file. We don't explicitly need it and our branched are protected so won't probably overwrite this. 

Removed this job.